tests/Integration/Behaviour/Features/Context/Domain/Product/CommonProductFeatureContext.php

```diff
@@ -305,7 +305,7 @@ public function assertIsIndexed(string $productReference): void
     }
 
     /**
-     * @Then product :productReference should be indexed for shops ":shopReferences"
```

Comment:
why you remove the "" ?
I usually prefer to keep them as in some cases it helps understanding where the reference ends. For example with "" you can provide a reference with spaces (like "my shop 1" , whereas without them you cannot)

Reply:
Because I noticed some bugs with the `""` like ti works fine with one shop or to "shop1,shop2" but weirdly with more it didn't match the step any more 🤷‍♂️ Removing the quotes sovled this problem so I didn't think too much 😅